### PR TITLE
Fix link to Sys.cpu_info

### PR DESCRIPTION
**Pull request by nalimilan**
This fixes building the Documenter docs. Introduced by https://github.com/JuliaLang/julia/pull/31204.

### CODE REVIEW

File: base/sysinfo.jl [hunk 1]
```diff
@@ -196,9 +196,10 @@ function _cpu_summary(io::IO, cpu::AbstractVector{CPUinfo}, i, j)
 end
 
 """
-    Sys.cpu_summary(io::IO=stdout, cpu::AbstractVector{CPUinfo} = cpu_info())
+    Sys.cpu_summary(io::IO=stdout, cpu::AbstractVector{CPUinfo} = Sys.cpu_info())
```
Comment (dkarrasch) [inline base/sysinfo.jl]: Perhaps we should remove the whitespaces in the second keyword argument? It took me quite a few seconds to parse that line and to see whether there is an assignment going on here...

Comment (nalimilan) [reply base/sysinfo.jl]: Good idea.

**Comment by staticfloat** [comment]
Good catch @nalimilan; I went the nuclear route and reverted the whole commit.  You'll need to rebase on top of that.

**Comment by nalimilan** [comment]
AFAICT there's nothing to rebase since the changes in this PR applied to a docstring which has been removed by the revert. They will just have to be applied to the un-reverting PR.

**Pull request closed without merging**